src/application/agora/widgets/proposal-main.container.tsx

```diff
@@ -1,40 +1,21 @@
-import { useCallback, useState } from 'react';
+import { useState } from 'react';
 
-import { ErrorBoundary } from 'shared/observability';
-import { useCommandQuery } from 'shared/messaging';
 import { Pagination } from 'shared/ui';
+import { ErrorBoundary } from 'shared/observability';
 
-import { useTwitterAgoraWidgetsInfo } from '../hooks';
-import { GetAgoraProposalsCommand } from '../commands';
+import { useProposalsQuery } from '../hooks';
 
 import { Proposal } from './proposal';
 
-export const ProposalMainContainer = () => {
-  const [hiddenAgoraUsers, setHiddenAgoraUsers] = useState<string[]>([]);
-  const [currentProposalIndex, setCurrentProposalIndex] = useState(0);
-
-  const { visibleAgoraWidgetsInfo: visibleAgoraNodes } =
-    useTwitterAgoraWidgetsInfo({
-      hidden: hiddenAgoraUsers,
-    });
-
-  const hideAgoraUser = useCallback((agoraUsername: string) => {
-    setHiddenAgoraUsers((previous) => {
-      return [...previous, agoraUsername];
-    });
-  }, []);
+interface Properties {
+  className?: string;
+  onClose?: () => void;
+}
 
-  const proposalQuery = useCommandQuery({
-    command: new GetAgoraProposalsCommand({
-      offset: currentProposalIndex,
-    }),
-    enabled: visibleAgoraNodes ? visibleAgoraNodes.length > 0 : false,
-    staleTime: Number.POSITIVE_INFINITY,
-    placeholderData: (previousData) => {
-      return previousData;
-    },
-  });
+const Base = ({ className, onClose }: Properties) => {
+  const [currentProposalIndex, setCurrentProposalIndex] = useState(0);
 
+  const proposalQuery = useProposalsQuery({ offset: currentProposalIndex });
   const currentProposal = proposalQuery.data?.proposal;
   const isLoadingProposal =
     proposalQuery.isLoading || proposalQuery.isPlaceholderData;
```

Comment:
I think we can get rid of isPlaceholder, since query don't use the placeholderData